examples/data-objects/webflow/src/clipboard/paste.ts

```diff
@@ -21,7 +21,8 @@ export function paste(doc: FlowDocument, data: DataTransfer, position: number) {
         const root = document.createElement("span");
         root.innerHTML = content;
         pasteChildren(doc, root, position);
-    } else if (content = data.getData(ClipboardFormat.html)) {
+        // ESLint found a bug!
+    } else if (content = data.getData(ClipboardFormat.text)) {
```

Comment:
I think ESLint found a bug here, and I made a fix. Is it the right one?

Reply:
Can you avoid making the bug fix in this PR, so it's isolated in history?

Reply:
Yes, good call. I'll revert this.

Reply:
PR #3836.